docs/guide/getting-started.md

```
@@ -4,17 +4,16 @@ outline: 2
 
 # Getting Started
 
 This guide will walk you through the steps to set up and use the `vitepress-openapi` in your VitePress project.
 
-## Demo
+## Showcase
 
-- [DolarApi.com](https://dolarapi.com/)
-- [ArgentinaDatos](https://argentinadatos.com/)
-- [CriptoYa API](https://docs.criptoya.com/)
+<ShowcaseList />
```

Markdownlint (0.37.0)

Element: ShowcaseList
Inline HTML

(MD033, no-inline-html)